v3/plugins/eclipse/che-theia/next/meta.yaml

```diff
@@ -78,10 +78,11 @@ spec:
          - exposedPort: 13131
          - exposedPort: 13132
          - exposedPort: 13133
-     memoryLimit: "512M"
+     memoryLimit: 512M
+     cpuLimit: 500m
```

Comment:
Does it really work well?
The benchmarks @vzhukovs did in https://github.com/eclipse/che/issues/18565#issuecomment-753686165 say the opposite - with 500m Che-Theia slows down significantly. See the `Measurement 3`.